torchci/lib/bot/cliParser.ts

```diff
@@ -187,7 +187,7 @@ parser.add_argument("-h", "--help", {
   action: "store_true",
 });
 
-const botCommandPattern = new RegExp(/^ *@pytorch(merge|)bot.*$/m);
+const botCommandPattern = new RegExp(/^ *@pytorch(merge|)bot .*$/m);
```

Comment:
Multiple spaces between at-mention and command should be OK as well, shouldn't it? (I.e. `@pytorchbot   help`)
```suggestion
const botCommandPattern = new RegExp(/^ *@pytorch(merge|)bot +.*$/m);
```

Reply:
No need for that, it's already handled by `.*`